R/partial_dependence.R

```diff
@@ -145,6 +145,29 @@ handle_partial_dependence <- function(x) {
   names(chart_type_map) <- colnames(df)
 
   ret <- ret %>%  dplyr::mutate(chart_type = chart_type_map[x_name])
+
+  # Sort the rows for scatter plots for categorical predictor variables, by Predicted values.
+  nested <- ret %>% dplyr::group_by(x_name, y_name) %>% tidyr::nest(.temp.data=c(-x_name)) #TODO: avoid possibility of column name conflict between .temp.data and group_by columns.
+  nested <- nested %>% dplyr::mutate(.temp.data = purrr::map(.temp.data, function(df){
+    if (df$chart_type[[1]]=="scatter" && "Predicted" %in% unique(df$y_name)) {
```

Comment:
But looks we need `y_name` here.

Reply:
Should comment that "Predicted" column is not available in Multiple classification case.